src/styles/blocks/contentCode.scss

```diff
@@ -0,0 +1,15 @@
+.contentCode {
+  margin-bottom: 16px;
+  color: #616070;
+  width: 98px;
+  height: 14px;
+  top: 247px;
+  left: 17px;
+  font-family: $main-font-famaly;
```

Comment:
There is a typo in the variable name '$main-font-famaly'. It should be '$main-font-family'. Ensure that the variable is correctly defined and imported.